pretix_sepadebit/signals.py

```diff
@@ -59,6 +70,65 @@ def register_csv(sender, **kwargs):
     return DebitList
 
 
+def get_todo_sepa_reminders(date):
+    q_list = []
+
+    for event in Event.objects.all():
+        if event.settings.payment_sepadebit__enabled:
+            prenotification_days = int(event.settings.payment_sepadebit_prenotification_days)
+            q_list.append(Q(
+                reminder=SepaDueDate.REMINDER_OUTSTANDING,
+                payment__provider='sepadebit',
+                date__lte=now().date() + timedelta(days=prenotification_days))
+            )
+
+    if len(q_list) > 0:
+        qs = reduce(or_, q_list)
+        return SepaDueDate.objects.filter(qs).select_related('payment')
+    return []
+
+
+@receiver(signal=periodic_task, dispatch_uid="payment_sepadebit_send_payment_reminders")
+def send_payment_reminders(sender, **kwargs):
+    with scopes_disabled():
+        dd = get_todo_sepa_reminders(date=now().date())
+
+        for due_date in dd:
+            order = due_date.payment.order
+            event = order.event
+            subject = LazyI18nString(event.settings.payment_sepadebit_mail_payment_reminder_subject)
+            text = LazyI18nString(event.settings.payment_sepadebit_mail_payment_reminder_text)
+
+            ctx = {
+                'event': event,
+                'order': order,
+                'creditor_id': event.settings.payment_sepadebit_creditor_id,
+                'creditor_name': event.settings.payment_sepadebit_creditor_name,
+                'account': due_date.payment.info_data['account'],
+                'iban': due_date.payment.info_data['iban'],
```

Comment:
Uh, one more thing. We should probably mask the IBAN and only send "DE23****4455" in the email, it's probably not a good idea to send unencrypted IBANs around without need.